Dockerfile

```diff
@@ -71,6 +71,4 @@ VOLUME $GERRIT_SITE
 
 ENTRYPOINT ["/gerrit-entrypoint.sh"]
 
-EXPOSE 8080 29418
-
```

Comment:
Any idea that we should remove this one?

Reply:
I have added `EXPOSE` back in, but it isn't required - [expose docs](https://docs.docker.com/engine/reference/builder/#expose).

> The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container...